include/openssl/base.h

```diff
@@ -188,7 +188,6 @@ extern "C" {
 
 #define AWSLC_VERSION_NAME "AWS-LC"
 #define OPENSSL_IS_AWSLC
-#define OPENSSL_VERSION_NUMBER 0x1010107f
 #define SSLEAY_VERSION_NUMBER OPENSSL_VERSION_NUMBER
```

Comment:
This changes behaviour. Everything that included `base.h` before would have `OPENSSL_VERSION_NUMBER` defined. Now it is scoped to only `crypto.h` (or `opensslv.h` directly).

And for this specific line: `SSLEAY_VERSION_NUMBER` is now undefined.

Reply:
Switched to your less intrusive suggestion.